### DISCUSSION TITLE
Is there a i3status placeholder for internet access

### DISCUSSION THREAD

**Discussion by gh-supersudo** [i3/i3#6284] [Q&A] [answered]
There's one if the device is connected to wlan.

```conf
format_up = " W: %essid (%quality ) "
```

But is there one if the device is connected to internet?

**Comment by okraits** [comment, marked as answer]
You could use Path-exists to implement that kind of functionality:

https://i3wm.org/i3status/manpage.html#_path_exists

- Have a script running which periodically checks for internet access (ping a specific host, etc.) and creates or deletes a file accordingly
- Use the Path-exists function to display the result via i3status
